src/main/java/me/jellysquid/mods/sodium/client/render/chunk/lists/ChunkRenderList.java

```diff
@@ -101,4 +103,14 @@ public RenderRegion getRegion() {
         return this.region;
     }
 
+    /**
+     * Get the squared distance of this region from the camera.
+     */
+    public double getDistanceFromCamera() {
```

Comment:
to avoid confusion calling this explicitly squared in the method name might be good. (same for the companion setter method and the field)